keyboards/ph_arisu/info.json

```diff
@@ -0,0 +1,291 @@
+{
+    "keyboard_name": "PH Arisu",
+    "url": "https://github.com/FateNozomi/arisu-pcb",
+    "maintainer": "qmk",
+    "width": 19.75,
+    "height": 5.25,
+    "layouts": {
+        "LAYOUT_default": {
+            "layout":
```

Comment:
This is not meant to contain the KLE layout. Please use https://qmk.fm/converter to convert the raw KLE JSON to the proper format.

Reply:
Many Thanks Updated info.json here https://github.com/qmk/qmk_firmware/pull/9612/commits/a38dea6148b5000ed2ffc9a34e14e87ffc69bebb